### DISCUSSION TITLE
find the error in this code why is it showing the error and do the needful corrections and please explain me with that

### DISCUSSION THREAD

**Discussion by KratiGupt** [Ajaysinh1290/College-Management-System-In-Java#6] [General]
lass Solution {
    public ListNode removeNthFromEnd(ListNode head, int n) {
        ListNode temp=head;
        int c=0;
        while(temp.next!=null){
            temp=temp.next;
            c++;
        }
        int length=c-n;
       
        while(c!=length){
            temp=temp.next;
            c--;
        }
        ListNode tem=temp.next;
        temp.next=temp.next.next;
        return temp;
    }
}

**Comment by Ajaysinh1290** [comment]
Hey KratiGupt, I do not find provided code in my project. can you ellaborate where this code in written. and if this code is not related to this project than please delete this discussion and mail me the problem with proper questions. Thanks
